x-pack/plugins/apm/public/components/app/service_groups/service_group_save/select_services.tsx

```diff
@@ -144,6 +151,21 @@ export function SelectServices({
                     setStagedKuery(value);
                   }}
                   value={kuery}
+                  suggestionFilter={(querySuggestion) => {
+                    if ('field' in querySuggestion) {
+                      const {
+                        field: {
+                          spec: { name: fieldName },
+                        },
+                      } = querySuggestion;
```

Comment:
nit: this is 100% personal preference but my brain always struggle a bit with nested destructurings. Feel free to leave as-is if you prefer that.

```suggestion
                      const fieldName = querySuggestion.field.spec.name;
```